hash_to_array.rb

```diff
@@ -0,0 +1,20 @@
+my_hash = {}
+
+5.times do |i|
+    puts "Enter #{i + 1} key name:"
+    temp_key = gets.chomp
+    puts "Enter #{i + 1} value data:"
+    temp_value = gets.chomp
+    my_hash.store(temp_key, temp_value)
```

Comment:
FYI it's far more conventional to use the square bracket syntax for this.

```suggestion
    my_hash[temp_key] = temp_value
```

Reply:
done